### DISCUSSION TITLE
Add analytics feature for supervision.

### DISCUSSION THREAD

**Discussion by MHD-HD98** [roboflow/supervision#1246] [Q&A]
Is it possible to add the new analytics feature released by ultralytics for the Dwell Time Analysis project?

**Comment by SkalskiP** [comment]
Hi @MHD-HD98, could you be a bit more specific?

**Reply by MHD-HD98** [reply]
In the dwell time analysis project i would like to obtain an analysis of number of people at a specific counter for a certain period of time. I want this analysis in the form of a bar graph.

[Code](https://docs.ultralytics.com/guides/analytics)  This repo contains how it's done in ultralytics.
